index.html

```diff
@@ -8,30 +8,236 @@
+                            <li class="js">Javascript</li>
+                            <li class="html">html</li>
+                        </ul>
+                        <div class="btn-container">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </article>
+            </section>    
+        </main>
+        <aside class="aside-container">
+            <section class="aside-flex">    
+                <div class="header-container-about">
+                    <h2 class="header header-2" id="about-head">About me</h2>
+                </div>
+                <div class="about-paragraph paragraph">
+                    I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
                     don’t hestiate to contact me.
-                </p>
-                <address>
-                    <ul class="container nav-container">
-                        <li class="nav-link-container"><a href="github.com/fobadara" class="nav-link"><img src="images/github.svg" alt="An image showing Github's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/linkedin-icon.svg" alt="An image showing Linkedin's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/angellist.svg" alt="An image showing angelist's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/twitter.svg" alt="An image showing twitter's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/medium.svg" alt="An image showing medium's logo"></a></li>
+                </div>
+                <div class="btn-container" id="resume-btn-container">
+                    <button type="button" class="btn" id="resume-btn">Get My Resume</button>
+                </div>
+            </section>     
+        </aside>
+        <div class="skillset">
+            <article class="all-skills">
+                <div class="box-container" id="lang-container">
+                    <div class="logo-container" id="lang-logo-container">
+                        <img src="images/lang.svg" alt="" class="type" id="lang-logo">
+                    </div>
+                    <div class="header-container">
+                        <h3 class="header" id="lang">Languages</h3>
+                    </div>
+                    <ul class="container code-skill" id="code-lang-list">
+                        <li class="lang">Javascript</li>
+                        <li class="lang">Ruby</li>
+                        <li class="lang">Html</li>
+                        <li class="lang">css</li>
                     </ul>
```

Comment:
The languages should be styled as shown below
#### Original design
![right](https://user-images.githubusercontent.com/58376020/126040953-c040ec4c-2ad8-48e4-9a0d-5aa83d652cce.png)
### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126040962-6097b92b-aa12-43e6-8504-a10a2eb592b6.png)

Reply:
I have done this too. It actually changed after I fixed some conflict